db/schema.rb

```diff
@@ -26,6 +26,8 @@
     t.datetime "updated_at", precision: 6, null: false
     t.string "status"
     t.json "offer"
+    t.datetime "rejected_at"
```

Comment:
@RobertM905 Looks like the domain model diagram wasn't updated with these changes, can you check if it works on your machine?

Reply:
I'm having problems running erd. `Failed: KeyError: key not found: "FIND_BASE_URL"